Extension/src/LanguageServer/settings.ts

```diff
@@ -165,14 +164,249 @@ export class CppSettings extends Settings {
+        return super.Section.get<string>("formatting.engine");
+    }
+
+    public get vcFormatIndentBraces() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.braces") === true
+    }
+    
+    public get vcFormatIndentMultiLineRelativeTo(): string | undefined {
+        return super.Section.get<string>("vcFormat.indent.multiLineRelativeTo");
+    }
+
+    public get vcFormatPreserveIndentationWithinParenthesis() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.preserveIndentationWithinParentheses") === true
+    }
+
+    public get vcFormatIndentCaseLabels() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.caseLabels") === true
+    }
+
+    public get vcFormatIndentCaseContents() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.caseContents") === true
+    }
+    
+    public get vcFormatIndentCaseContentsWhenBlock() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.caseContentsWhenBlock") === true
+    }
+    
+    public get vcFormatIndentLambdaBracesWhenParameter() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.lambdaBracesWhenParameter") === true
+    }
+    
+    public get vcFormatIndentGotoLables() :  string | undefined {
+        return super.Section.get<string>("vcFormat.indent.gotoLabels");
+    }
+
+    public get vcFormatIndentPreprocessor() :  string | undefined {
+        return super.Section.get<string>("vcFormat.indent.preprocessor");
+    }
+
+    public get vcFormatIndentAccessSpecifiers() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.accessSpecifiers") === true
+    }
+
+    public get vcFormatIndentNamespaceContents() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.namespaceContents") === true
+    }
+
+    public get vcFormatIndentPreserveComment() : boolean {
+        return super.Section.get<boolean>("vcFormat.indent.preserveComment") === true
+    }
+
+
+    public get vcFormatNewlineBeforeOpenBraceNamespace() : string | undefined {
+        return super.Section.get<string>("vcFormat.newLine.beforeOpenBrace.namespace");
+    }
+
+    public get vcFormatNewlineBeforeOpenBraceType() : string | undefined {
+        return super.Section.get<string>("vcFormat.newLine.beforeOpenBrace.type");
+    }
+
+    public get vcFormatNewlineBeforeOpenBraceFunction() : string | undefined {
+        return super.Section.get<string>("vcFormat.newLine.beforeOpenBrace.function");
+    }
+
+    public get vcFormatNewlineBeforeOpenBraceBlock() : string | undefined {
+        return super.Section.get<string>("vcFormat.newLine.beforeOpenBrace.block");
+    }
+
+    public get vcFormatNewlineBeforeOpenBraceLambda() : string | undefined {
+        return super.Section.get<string>("vcFormat.newLine.beforeOpenBrace.lamda");
+    }
+
+    public get vcFormatNewlineScopeBracesOnSeparateLines() : boolean | undefined {
```

Comment:
Using `=== true` removes the potential for an `undefined` result.  You'll want to remove ` | undefined` from the return type of these functions that use `=== true`.